src/content/developers/docs/scaling/validium/index.md

```diff
@@ -7,30 +7,64 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-Uses validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/) but data is not stored on the main layer 1 Ethereum chain. This can lead to 10k transactions per second per validium chain and multiple chains can be run in parallel.
+Validium is a [scaling solution](/developers/docs/scaling/) that enforces integrity of transactions using validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/), but doesn’t store transaction data on the main Ethereum layer. While off-chain data availability introduces trade-offs, it can lead to massive improvements in scalability (validiums can process ~ 9,000 transactions, or more, per second). 
 
 ## Prerequisites {#prerequisites}
 
 You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as Validium is an advanced topic as the technology is less battle-tested, and continues to be researched and developed.
 
-## Pros and cons {#pros-and-cons}
+## How does validium work? {#how-does-validium work}
 
+Validiums are a type of “rollup”: a [Layer 2](/layer-2/) scaling protocol designed to improve throughput by processing transactions off the main Ethereum chain. Off-chain transactions executed on the validium chain are verified via a smart contract on the base Ethereum layer using zero-knowledge proofs. This can be either a SNARK (Succinct Non-Interactive Argument of Knowledge) or SNARK (Scalable Transparent Argument of Knowledge). 
```

Comment:
```suggestion
Validiums are a type of “rollup”: a [Layer 2](/layer-2/) scaling protocol designed to improve throughput by processing transactions off the main Ethereum chain. Off-chain transactions executed on the validium chain are verified via a smart contract on the base Ethereum layer using zero-knowledge proofs. This can be either a SNARK (Succinct Non-Interactive Argument of Knowledge) or STARK (Scalable Transparent ARgument of Knowledge). 
```

Reply:
Might be nice to link out to learning resources for SNARK & STARK.

Reply:
e.g. https://consensys.net/blog/blockchain-explained/zero-knowledge-proofs-starks-vs-snarks/